.travis.yml

```diff
@@ -24,14 +23,14 @@ email:
 jobs:
   include:
     # Tests ---
-    - &test-full
-      stage: test-full
+    - &test-extended
+      stage: test-extended
       before_script:
         - curl -L https://codeclimate.com/downloads/test-reporter/test-reporter-latest-linux-amd64 > ./cc-test-reporter
         - chmod +x ./cc-test-reporter
         - ./cc-test-reporter before-build
       script:
-       - python setup.py test
+        - python setup.py test --pytest-args='-o python_classes=ExtendedTests'
```

Comment:
Did you check that all the tests are either in BasicTests or ExtendedTests?

Reply:
yup, Tests which are neither in BasicTests nor in ExtendedTests (e.g. 'unit/test_madx_wrapper.py') are run in both cases